app/js/models/addon_generator.js

```diff
@@ -126,25 +126,51 @@ el.parentNode.removeChild(el);
     );
   }
 
-  opSetAttribute(name, value) {
+  opCreateAttribute(expression, name) {
```

Comment:
This isn't a real operation, it's just a helper function. It would be best to name it that way.